### ISSUE TITLE
TypeError: PlayCommand.createResponse is not a function

### ISSUE THREAD

**Issue by Alan-Wen-3344** [galnir/Master-Bot#523]
Hello,
When I try to do `!np` it shows this error message: An error occurred while running the command: `TypeError: PlayCommand.createResponse is not a function`. I've no idea if this is a node bug or something?
Tried with node v12.22 and v14.16 but both of them are having the same issue, is there a way to fix it?

**Comment by galnir** [comment]
Did you clone this bot or did you copy code?
PlayCommand.createResponse is an old function in play.js (which was rewritten not long ago)

**Comment by Alan-Wen-3344** [comment]
Yes I use git bash to git clone~
Also didn't modify anything...

**Comment by galnir** [comment]
Your clone is out of date because as I wrote there is no PlayCommand.createResponse anymore

**Comment by TheRedyYT** [comment]
> Your clone is out of date because as I wrote there is no PlayCommand.createResponse anymore

The nowplaying command use PlayCommand.createResponse
```
const { Command } = require('discord.js-commando');
const PlayCommand = require('./play');

module.exports = class NowPlayingCommand extends Command {
  constructor(client) {
    super(client, {
      name: 'nowplaying',
      group: 'music',
      memberName: 'nowplaying',
      aliases: ['np', 'currently-playing', 'now-playing'],
      guildOnly: true,
      description: 'Display the currently playing song!'
    });
  }

  run(message) {
    if (
      (!message.guild.musicData.isPlaying &&
        !message.guild.musicData.nowPlaying) ||
      message.guild.triviaData.isTriviaRunning
    ) {
      message.reply(':no_entry: Please join a voice channel and try again!');
      return;
    }

    PlayCommand.createResponse(message)
      .setColor('#6c6c6c')
      .build();
  }
};
```

**Comment by galnir** [comment]
@TheRedyYT My bad, it's indeed a bug I forgot to solve when I rewrote play.js
I'll look into solving it asap

**Comment by galnir** [comment]
Solved in cbc13af

Thank you @Alan-Wen-3344 for letting me know about this. You're always welcome to report bugs or suggest new features :)
